azure-functions/acbs-function/api.js

```diff
@@ -1,75 +1,93 @@
 /**
- * ACBS Functions API Library deals with following HTTP Methods:
+ * ACBS durable functions API library deals with following HTTP methods:
+ *
  * 1. GET
  * 2. POST
  * 3. PUT
  * 4. PATCH
  *
- * All the function have argument validation check and return object verification in
+ * All of the function have argument validation check and return object verification in
  * case err object does not have expected properties due to network connection, SSL verification or other issues.
  */
-const acbs = process.env.MULESOFT_API_UKEF_TF_EA_URL;
-const username = process.env.MULESOFT_API_KEY;
-const password = process.env.MULESOFT_API_SECRET;
-const axios = require('axios');
 
 require('dotenv').config();
+const axios = require('axios');
+
+const headers = {
+  [String(process.env.APIM_TFS_KEY)]: process.env.APIM_TFS_VALUE,
+  'Content-Type': 'application/json',
+};
+const badRequest = {
+  status: 400,
+  data: {},
+};
+
+/**
+ * Invokes TFS GET endpoint
+ * @param {String} endpoint TFS endpoint
+ * @returns {Object} API response object
+ */
+const get = async (endpoint) => {
+  if (endpoint) {
+    const url = `${process.env.APIM_MDM_URL}/${endpoint}`;
 
-const getACBS = async (apiRef) => {
-  if (apiRef) {
     return axios({
       method: 'get',
-      url: `${acbs}/${apiRef}`,
-      auth: {
-        username,
-        password,
-      },
-      headers: {
-        'Content-Type': 'application/json',
-      },
-    }).catch((e) => {
-      console.error('Error calling GET to ACBS');
+      url,
+      headers,
+    }).catch((error) => {
+      console.error('Error calling ACBS GET /%s', endpoint);
```

Comment:
`/%s` typo?

Reply:
More instances below - if this is intended what does it mean?

Reply:
This is intended it means it would print the endpoint with leading `/` i.e. `/deals'.

Reply:
Is it worth using this style of string interpolation instead: 
```
console.error(`Error calling ACBS GET /${endpoint}`);
```
It looks like it's used elsewhere in this file and might avoid confusion in future?